nats-server doen´t build on OpenBSD 6.9

Hi, I tried to use eats-server on OpenBSD 6.9 in ran the `go get` command:

ud$ go version 
go version go1.16.2 openbsd/amd64
ud$ GO111MODULE=on go get github.com/nats-io/nats-server/v2  
go: downloading github.com/nats-io/nats-server/v2 v2.3.4
go: downloading github.com/nats-io/jwt/v2 v2.0.3
# github.com/nats-io/nats-server/v2/server
go/pkg/mod/github.com/nats-io/nats-server/v2@v2.3.4/server/disk_avail.go:31:23: fs.Bavail undefined (type syscall.Statfs_t has no field or method Bavail)
go/pkg/mod/github.com/nats-io/nats-server/v2@v2.3.4/server/disk_avail.go:31:43: fs.Bsize undefined (type syscall.Statfs_t has no field or method Bsize)
ud$

--- a/server/disk_avail.go
+++ b/server/disk_avail.go
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !windows 
+// +build !windows
 // +build !openbsd
 
 package server

--- a/server/disk_avail_openbsd.go
+++ b/server/disk_avail_openbsd.go
@@ -1,4 +1,4 @@
-// Copyright 2020 The NATS Authors
+// Copyright 2021 The NATS Authors
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build openbsd 
+// +build openbsd
 
 package server
 

